packages/frontend/src/common/components/Table.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import TableColumnTitle from '@/common/components/TableColumnTitle'
+import TableRow from '@/common/components/TableRow'
+import TableIconRow from '@/common/components/TableIconRow'
+import { Entity } from '@/common/interfaces'
+import { TableCellType } from '@/common/enums/TableCellType'
+
+interface Params {
+  tableDataConfig: TableDataConfigItem[]
+  items: Entity[]
+}
+
+export interface TableDataConfigItem {
+  title: string
+  additionalTitleClasses?: string
+  columnValue: string
+  cellType?: TableCellType
+}
+
+const Table = ({ tableDataConfig, items }: Params) => {
+  return (
+    <div className="overflow-x-auto">
+      <table className="table">
+        <thead>
+          <tr className="text-sm text-primary">
+            {tableDataConfig.map(
+              (tableDataItem: TableDataConfigItem, index: number) => (
+                <TableColumnTitle
+                  key={index}
+                  additionalTitleClasses={tableDataItem.additionalTitleClasses}
+                  title={tableDataItem.title}
+                />
+              ),
+            )}
+          </tr>
+        </thead>
+        <tbody>
+          {items.map((item: any, rowIndex: number) => {
+            return (
+              <tr key={rowIndex}>
+                {tableDataConfig.map(
+                  (
+                    tableDataItem: TableDataConfigItem,
+                    cellIndex: number,
+                    array: TableDataConfigItem[],
+                  ) => {
+                    let additionalRowClasses: string = ''
+
+                    if (cellIndex === 0) {
+                      additionalRowClasses += 'rounded-l-lg'
+                    } else if (cellIndex + 1 === array.length) {
+                      additionalRowClasses += 'rounded-r-lg'
```

Comment:
Do you not want the top and bottom to be rounded? Not just left / right?

Reply:
Or if it's specific to a cell, not row, perhaps the naming should be related to cell, i.e. `additionalCellClasses`

Reply:
Yes, it's specific to a first and last cell. 
`rounded-l-lg` is for top and bottom from left side  and `rounded-r-lg` for both of right side.
Parameter was removed together with abstraction.